File: group-grpc/202000211/demo-server-01/src/main/java/luxe/chaos/demo/controller/EchoController.java
package luxe.chaos.demo.controller;

import luxe.chaos.demo.DemoServer01App;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Collections;
import java.util.Map;

@RestController
public class EchoController {

    private static final Logger LOGGER = LoggerFactory.getLogger(EchoController.class);


    @GetMapping(value = "/v1/say-hai")
    public Map<String, Object> sayHello() {

        return Collections.singletonMap("message", "嗨..");
    }
}

File: ctripcorp-apollo/empty-spring-boot/src/main/scala/com/example/demo/controller/EchoController.scala
package com.example.demo.controller

import org.springframework.web.bind.annotation.{GetMapping, RestController}

@RestController
class EchoController {


  @GetMapping(value = Array("/echo"))
  def echo(name: String = "world") : String = {
    s"hello $name"
  }


}
